client_app/src/App.js

```diff
@@ -1,14 +1,38 @@
-import {UpcomingShifts} from "./Shifts";
+import { BrowserRouter as Router, Routes, Route, Link } from "react-router-dom";
+
+import logo from "./Assets/gethelplogo.png";
+
+import { PastShifts, UpcomingShifts } from "./Shifts";
 import Shelters from "./Shelters";
+import VolunteerDashboard from "./VolunteerDashboard";
+import ShelterDashboard from "./ShelterDashboard";
+
 import "./App.css";
 
 function App() {
   return (
     <div>
-        {UpcomingShifts()};
-        {Shelters()};
+      <Router>
+        <div class="main-nav">
+          <Link to="/">
+            <img class="logo" src={logo} alt="GetHelp Logo" />
+          </Link>
```

Comment:
Add a volunteer dashboard link first, which takes the user to the main page

Reply:
@kate-holdener Since I currently have the volunteer dashboard as the main page, do you want me to add another link that redirects to the main page? Or is the volunteer dashboard going to be separate from the main page?

Reply:
@loganwyas Sorry, my comment was confusing. What I meant was: the main page should be the volunteer dashboard. The navigation menu should have an option to get back to the dashboard. 

It didn't seem right that the GetHelp logo took the user to our main page (seems like it should take the user to the GetHelp main page). Or maybe we need a different logo, which could be the link to the dashboard.